### DISCUSSION TITLE
Can't get article link shortcode to work.

### DISCUSSION THREAD

**Discussion by monstermuffin** [nunocoracao/blowfish#500] [Q&A]
I am trying to use the `article link` shortcode in a few posts and I cannot get any content to appear regardless of the entered link.

My structure is as follows:

```
muffinsblog/
├─ content/
│  ├─ posts/
│  │  ├─ this-is-post-1/
│  │  │  ├─ index.md
│  │  ├─ this-is-post-2/
│  │  │  ├─ index.md

```

I have tried the following in `this-is-post-1`'s `index.md`:

```
{{< article link="posts/this-is-post-2" >}}
{{< article link="/this-is-post-2" >}}
{{< article link="posts/this-is-post-2/" >}}
{{< article link="/this-is-post-2/" >}}
{{< article link="/this-is-post-2/index.md" >}}
{{< article link="this-is-post-2" >}}
{{< article link="content/posts/this-is-post-2" >}}
{{< article link="content/posts/this-is-post-2.md" >}}
{{< article link="posts/this-is-post-2/index.md" >}}
```

I would expect one of these variations to work but I can't get the shortcode to work. 

Thanks.

**Comment by nunocoracao** [comment]
What is the baseurl for the website?

**Reply by monstermuffin** [reply]
blog.muffn.io

**Reply by nunocoracao** [reply]
try "/posts/adding-goatcounter-to-hugo/"

**Reply by monstermuffin** [reply]
> try "/posts/adding-goatcounter-to-hugo/"

This worked. I could swear I tried this format but clearly not. 

Thank you!
